### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by finch38** [ps882/ORIE4741-Project#8]
The project problem seemed complicated at first, but the project was very interesting to read.  The dataset description was well written resolving all (at least I think all) possible factors that could impact the model's performance through feature engineering.  The implementation of the different models, both learned in class and outside of class, were also well-detailed and very reasonable.  I enjoyed reading the different implementations used to solve the gender classification problem.  

I'm a little surprised about how accurate your model is (0.981) yet you think there is room for improvement.  I would suggest bootstrapping your dataset to see if you come up with varying success rates.  GLRM would be interesting to implement on this dataset if PCA didn't work out.  Also, you mention overfitting Random foresting, but I don't have anything quantitative to look at to conclude overfitting.  I would suggest writing down the MSE scores of all the models (both train, validation, and test sets) and comparing them in the paper.  Are the success rates mentioned from the validation set or test set?

**Comment by sivasankalpp** [comment]
Thank you so much for your detailed review! 
- We believe we can do better because of the fact that the data set size we used was relatively small. With more samples, better features, and more sophisticated models, we believe we can push the score to 0.99. ;) 
- The issue with PCA is it is affected by scale. If we normalize our columns (features), PCA might still work. 
- Regarding Random Forest, we mentioned that are NOT likely to overfit as they do bootstrapping (as you mentioned) under the hood. 
- As we mentioned in the first line on page 7 (under Model Analysis), the success rates we mentioned are all on the test set. The Python notebook also contains more scores and graphs. 

Thank you!
